packages/donate-button-v4/src/autoPlayMode.tsx

```diff
@@ -0,0 +1,178 @@
+import {render as preactRender} from 'preact';
+import EmbedButton from 'src/components/embed-button';
+import {WidgetConfig} from 'src/components/widget/types/WidgetConfig';
+import {loadFonts} from 'src/loadFonts';
+import {WidgetLoader} from 'src/loaders/Widgetloader';
+import resetcss from 'src/resetCss';
+
+const OBSERVER_OPTIONS = {
+	childList: true,
+	subtree: true
+};
```

Comment:
```suggestion
    attributeFilter: ['href']
};
```
We only need to know if `href` changed